roles/provision-metrics-apb/tasks/provision-grafana.yml

```diff
@@ -0,0 +1,161 @@
+
```

Comment:
@StevenTobin is this a straight copy/paste from what was in the other file?

Reply:
Yes it is, I didn't make any changes.